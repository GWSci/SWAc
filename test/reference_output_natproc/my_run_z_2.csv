DATE,nDays,Area,Precipitation,PEt,VegetationPE_PEfac,CanopyStorage,PEfacLessCanopy_AE,Precipitation_Groundlevel,Snowfall,Precipitation_Rainfall,SnowPack,SnowMelt,Rainfall_SnowMelt,RapidRunoff,RunoffRecharge,MacroPoreAttenuated,MacroPoreDirect,Percolation_RootZone,RAWREW,TAWTEW,pSMD,SMD,AE,RejectedRecharge,PercolationThroughRootZone,SubRootZoneLeakege,BypassingInterflow,InputInterflowStore,InterflowStoreVol,InfiltrationRecharge,InterflowtoSW,InputRechargeStore,RechargeStoreVol,CombinedRecharge,SWAttenInputPot,SWAttenuation,PondDirect,PondAtten,OpenWaterAE,SWAttenInputAct,PondOverspill,OtherSW,OpenWaterPE,SWAbstractions,SWDischarges,CombinedSW,CombinedAE,UnitilisedPE,AVERAGE_IN,AVERAGE_OUT,TOTAL_STORAGE_CHANGE,BALANCE
05/01/1980,5,424.5785,3.17758795185,8.012207177419361,3.204882870967745,0.028598291566650034,3.1731069914492447,2.8312308650983504,2.187260200938578,0.9585518713929225,16.295222878588344,44.57484152352073,45.53339339491365,6.742604925808834,0.6609686852941848,0.0,0.0,34.23744912961346,269.713492125,490.38816750000007,43.8087332370124,75.47596570355196,2.570216663073889,0.0,0.0,0.106144625,0.004776508125000002,0.09123130518750001,13.030847175541668,1.3374222750000002,0.5846712450270836,2.1517699434191844,7.1218668637106495,0.3560933431855325,0.28798803342104295,2.5359089624831066,0.0,0.0,0.19717411746442842,0.2980462407953244,0.21942960604296316,0.26155365040574335,0.20723232483870982,5.174151508628476,0.0,-465.1105792040279,2.795989072104967,0.28557962923043206,3.28373257685,-461.9584967887375,31.311121823780255,433.9311075418072
30/09/2014,12687,424.5785,17851.095614314185,18225.77610316648,16298.508244870629,158.9163211912491,16121.934554658132,15907.06973169152,9164.13836114353,8510.383562958155,1207656.3394100557,9164.208629820943,17674.592192779102,3121.9760247652835,256.6941436825276,90.24259996655256,60.16173331103505,12634.752615458321,684371.0149179748,1244310.9362145,684301.594671232,690871.6592731742,6064.688013516005,4681.622994939814,1966.8384637125973,269.331371475,110.4618349020049,2109.8210466282935,23658.97494383298,977.2623441089643,1134.085629986201,1543.2456275610455,30899.887466802145,1605.1561066511424,3020.657564644157,8148.5172717515325,0.0,0.0,242.43105985643288,3048.181913440325,2704.3356919695393,343.5438637710485,269.95540865260074,6487.234689480074,0.0,-2738990.0057547796,6466.035394563686,8445.053085676313,18120.426985789185,-2730918.8142535645,-163.02174000978877,2749202.2629793636
