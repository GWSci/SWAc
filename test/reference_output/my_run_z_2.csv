DATE,nDays,Area,Precipitation,PEt,VegetationPE_PEfac,CanopyStorage,PEfacLessCanopy_AE,Precipitation_Groundlevel,Snowfall,Precipitation_Rainfall,SnowPack,SnowMelt,Rainfall_SnowMelt,RapidRunoff,RunoffRecharge,MacroPoreAttenuated,MacroPoreDirect,Percolation_RootZone,RAWREW,TAWTEW,pSMD,SMD,AE,RejectedRecharge,PercolationThroughRootZone,SubRootZoneLeakege,BypassingInterflow,InputInterflowStore,InterflowStoreVol,InfiltrationRecharge,InterflowtoSW,InputRechargeStore,RechargeStoreVol,CombinedRecharge,SWAttenuation,SWAbstractions,SWDischarges,CombinedSW,CombinedAE,UnitilisedPE,AVERAGE_IN,AVERAGE_OUT,TOTAL_STORAGE_CHANGE,BALANCE
01/01/1980,1,424.5785,0.0,1.6024414354838723,0.640976574193549,0.0,0.640976574193549,0.0,0.0,0.0,10.469988538744108,31.98786146125589,31.98786146125589,6.397572292251178,0.0,0.0,0.0,25.590289169004713,53.942698424999996,98.07763350000002,17.50853740518884,42.45785000000001,0.640976574193549,0.0,0.0,0.021228925,0.0010614462500000003,0.020167478750000002,3.7511510475000005,0.29720495,0.17269730487500004,0.29826639624999995,0.4245785,0.021228925,0.6325830968486483,4.589649274297351,0.0,1.4760272259801797,0.640976574193549,0.0,0.021228925,6.727881999471078,24.776615289936164,-31.48326836440724
30/09/2014,12691,424.5785,17854.27320226603,18232.18586890842,16301.072151167404,176.60546609201748,16124.466685075387,17677.66773617402,9166.325621344467,8511.342114829547,1207662.1646443955,9176.795609883207,17688.13772471276,3436.3250847419285,0.0,98.1026551378211,65.40177009188073,14088.308214741128,684586.785711675,1244703.2467485,784500.8192008927,791544.9090397626,7044.2183758713845,5051.378418929441,2070.3098740716373,269.41628717500004,116.98630806233187,2222.739853184305,24536.283671316298,1051.796756222225,1174.2243457547038,1266.8857194223779,25342.42853642189,1332.523196912975,1264.1597402269338,6497.245076263502,0.0,-2739429.1700141765,7220.823841963401,9080.248309204004,18123.689489441032,-2724378.5778990365,-81.7475467202383,2742584.014935198
